app/src/main/res/values-it/strings.xml

```diff
@@ -507,6 +507,6 @@
     <string name="change_sysinfo_art">Cambia l\'arte di \'sysinfo\'</string>
     <string name="sysinfo_art_change_description">Incolla qui l\'arte ASCII che vuoi visualizzare nel comando \'sysinfo\'.</string>
     <string name="sysinfo_art_updated">Arte ASCII aggiornata per il comando \'sysinfo\'!</string>
-    <string name="search">Search</string>
-    <string name="auto_execute_last_secondary_suggestion">Auto Execute last Secondary Suggestion</string>
+    <string name="search">Cerca</string>
+    <string name="auto_execute_last_secondary_suggestion">Esegui automaticamente l'ultimo Suggerimento Secondario</string>
```

Comment:
An apostrophe needs to be escaped with a '\'
Would have done it but I am not on my laptop rn.

Reply:
Corrected